checkpoint_schedules/schedule.py

```diff
@@ -19,48 +21,71 @@
 
 
 class StorageType(Enum):
-    """
-    RAM : It is the first type of checkpoint storage.
+    """ Enums to indicate the storage type of the checkpoint data.
+
+    RAM : A enum to indicate the storage of the forward checkpoint data in memory.
 
-    DISK : It is the second type of checkpoint storage.
+    DISK : A enum to indicate the storage of the forward checkpoint data in RAM.
     
-    WORK : ...
+    FWD_RESTART : A enum to indicate the storage of the forward checkpoint data
+    in a local used to initilise the forward solver.
 
-    NONE : Indicate that there is no specific storage location defined 
+    ADJ_DEPS : A enum to indicate the storage of the forward checkpoint data
+    in a local used for the adjoint computation.
```

Comment:
Forward restart and adjoint dependency data are not in general distinct, and so these should not refer to different storage locations